artifactory/services/upload.go

```diff
@@ -886,12 +889,17 @@ func (us *UploadService) CreateUploadAsZipFunc(uploadResult *utils.Result, targe
 
 // Reads files and streams them as a ZIP to a Reader.
 // archiveDataReader is a ContentReader of UploadData items containing the details of the files to stream.
+// isZipDryRun - If true, this function is run as part of the dry-run process, to
 // saveFilesPathsFunc (optional) is a func that is called for each file that is written into the ZIP, and gets the file's local path as a parameter.
-func (us *UploadService) readFilesAsZip(archiveDataReader *content.ContentReader, progressPrefix string, flat, symlink bool,
+func (us *UploadService) readFilesAsZip(archiveDataReader *content.ContentReader, isZipDryRun, flat, symlink bool,
```

Comment:
isZipDryRun --> dryRun

In addition to removing the redundant "zip" section from the argument name, that is unnecessary beucase the function's content is already related to the zip, it's preferred to avoid adding the "is" prefix to boolean variables.

Let's change the argument's description as well.

Reply:
@eyalbe4 its on purpose. We already have a dryRun expression among JFrog CLI command so I didn't want to confuse with that known flag.
Do you have another idea?